### DISCUSSION TITLE
🚀 FR horizontal scroll bar always visible- especially in grids

### DISCUSSION THREAD

**Discussion by robinsowell** [ExpressionEngine/ExpressionEngine#3736] [Feature Requests]
### Suggest an idea for ExpressionEngine

If you have a grid with a lot of rows, it's annoying to have to scroll to the bottom to then scroll across.  The request is for something like excell, where the scrollbar is always visible.  Even better if the column labels could always be visible as well.

There are a number of places this could be useful- I run into it all the time in the SQL manager looking at data that spans a lot of columns.  So really, anywhere this can happen, having the horizontal scroll always visible would make things much nicer.

**Comment by ciaran-framework-design** [comment]
Is this in the pipeline?

**Reply by robinsowell** [reply]
It is- at least for girds set to horizontal.
